backend/src/main/java/com/htecgroup/skynest/service/impl/FolderServiceImpl.java

```diff
@@ -113,6 +117,15 @@ public List<FolderResponse> getAllFoldersWithParent(UUID parentFolderId) {
     return asFolderResponseList(allFolders);
   }
 
+  @Override
+  public StorageContentResponse getFolderContent(UUID folderId) {
```

Comment:
You should check if the folder id exists, this way it returns an empty list for an invalid folder id. Some exception should be thrown